keda/values.yaml

```
@@ -254,6 +254,8 @@ priorityClassName: ""
 ## reasonable default
 http:
   timeout: 3000
+  keepAlive:
+    enabled: false
```

This should be `true` as default to maintaining current behaviour. When this is `true`, we don't add the env variable, so we don't disable the keep alive

```suggestion
    enabled: true
```